packages/services/src/services/Locale/Locale.js

```diff
@@ -114,8 +113,8 @@ class LocaleAPI {
       const lang = root.document.documentElement.lang.toLowerCase();
       const codes = lang.split('-');
       return { cc: codes[1], lc: codes[0] };
-    } else {
-      return _localeDefault;
+      //    } else {
+      //      return _localeDefault;
```

Comment:
I see we didn't remove `_localeDefault` is that going to cause any linting errors?